pillar.example

```diff
@@ -1,18 +1,17 @@
 postgres:
-  # Set True to configure upstream postgresql.org repository for YUM or APT
+  ## USE DISTRO REPO
   use_upstream_repo: False
-  # Version to install from upstream repository
-  version: '9.3'
+  service: postgresql
 
-  # These are Debian/Ubuntu specific package names
-  pkg: 'postgresql-9.3'
-  pkg_client: 'postgresql-client-9.3'
+  ## OR UPSTREAM REPO (Suse/Redhat)
+  # use_upstream_repo: True
+  # version: '9.5'
+  # service: postgresql-9.5
 
+  ## AND OVERRIDES
   # Additional packages to install with PostgreSQL server,
   # this should be in a list format
-  pkgs_extra:
-    - postgresql-contrib
-    - postgresql-plpython
+  pkgs_extra: []
```

Comment:
I'm aware that hard-coding platform-specific packages here was bad idea. But this possibly may affect later installation of the `uuid_ossp` extension defined below. Have you tried to run test kitchen with provided config?

Reply:
No, I've tested on OpenSUSE LEAP 42.

Reply:
Better to test your changes on other platforms as well, because you change shared things, and `pillar.example` has been used in existent test. That's not perfect, and the change is actually meaningful, but we need to understand an impact.

Reply:
I've already been testing formula on other distributions - see #171 #170 #164.

Only regression is #172 which is something you mentioned regarding pkgs_extra()

Reply:
I assume this affects Ubuntu and the existing test based on it.

Reply:
Also this is just a `pillar.example`, just leave the values as is to help people understand layout. We only get one pillar.example and support (unofficially) any number of distros so it doesnt need to change with every iteration of what distro was last used against it.

Reply:
That's exactly what I've been trying to express 👍 But having some working example that fits the most popular ones would be great.

Reply:
Agreed. Will action.